tests/agent_server/deploy/e2e/test_autostarted.py

```diff
@@ -688,6 +688,13 @@ def wait_for_new_executor() -> bool:
     for children in current_resumed_children.values():
         assert children.is_running()
 
+    result = await client.get_agents(environment)
+    assert result.code == 200
+    actual_data = result.result["data"]
+    assert len(actual_data) == 2
+    assert all([not agent["paused"] for agent in actual_data])
+    assert {e["name"] for e in actual_data} == {const.AGENT_SCHEDULER_ID, "agent1"}
```

Comment:
1. I don't think we want to show the `AGENT_SCHEDULER_ID` 
2. I think we should try to set a meaningfull up/down  state (e.g. `up iff scheduler.state  == UP and not agent.paused`